pkg/operator/configobservation/images/observe_images.go

```diff
@@ -41,14 +34,19 @@ func ObserveInternalRegistryHostname(genericListers configobserver.Listers, reco
 		return observedConfig, errs
```

Comment:
shouldn't we return `existingConfig` in this case ?

Reply:
it's not the normal error case here, but just the default value: empty observedConfig.

Reply:
consider adding a comment.